pkg/app/mysql.go

```diff
@@ -0,0 +1,228 @@
+	stdout, stderr, err := kube.Exec(mdb.cli, mdb.namespace, podname, containername, selectRowsCMD, nil)
+	format.Log(podname, containername, stdout)
+	format.Log(podname, containername, stderr)
+	if err != nil {
+		log.WithError(err).Print("Error while selecting rows from table.", field.M{"app": "mysql"})
+		return 0, err
+	}
+	// get the returned cound and convert it to int, to return
+	rowsReturned, err := strconv.Atoi((strings.Split(stdout, "\n")[1]))
+	return rowsReturned, nil
+}
+
+func (mdb *MysqlDB) Reset(context.Context) error {
+	log.Print("Resetting the mysql instance.", field.M{"app": "mysql"})
+	// delete all the data from the table
+	podname, containername, err := mdb.GetPodAndContainerName(mdb.namespace)
+	deleteFromTableCMD := []string{"sh", "-c", "mysql -u root --password=" + mdb.password + " -e 'DROP DATABASE IF EXISTS testdb'"}
+	stdout, stderr, err := kube.Exec(mdb.cli, mdb.namespace, podname, containername, deleteFromTableCMD, nil)
+	format.Log(podname, containername, stdout)
+	format.Log(podname, containername, stderr)
+	if err != nil {
+		log.WithError(err).Print("Error while dropping table.", field.M{"app": "mysql"})
+		return err
+	}
+
+	// create the database and a pets dummy table
+	createTableCMD := []string{"sh", "-c", "mysql -u root --password=" + mdb.password + " -e 'create database testdb; use testdb;  CREATE TABLE pets (name VARCHAR(20), owner VARCHAR(20), species VARCHAR(20), sex CHAR(1), birth DATE, death DATE);'"}
+	stdout, stderr, err = kube.Exec(mdb.cli, mdb.namespace, podname, containername, createTableCMD, nil)
+	format.Log(podname, containername, stdout)
+	format.Log(podname, containername, stderr)
+	if err != nil {
+		log.WithError(err).Print("Error while creatig table, in RESET table.", field.M{"app": "mysql"})
+		return err
+	}
+
+	return nil
+}
+
+func (mdb *MysqlDB) ConfigMaps() map[string]crv1alpha1.ObjectReference {
+	return nil
+}
+
+func (mdb *MysqlDB) Secrets() map[string]crv1alpha1.ObjectReference {
+	return map[string]crv1alpha1.ObjectReference{
+		"mysql": crv1alpha1.ObjectReference{
+			Kind:      "Secret",
+			Name:      "mysql-test",
+			Namespace: mdb.namespace,
+		},
+	}
+}
+
+// GetPodAndContianerName takes namespace as input and returns the pod and container that is running in
+// that namespace for deployment that was created through helm
+func (mdb *MysqlDB) GetPodAndContainerName(deploymentNamespace string) (string, string, error) {
+	deployment, err := mdb.cli.AppsV1().Deployments(mdb.namespace).Get("mysql-test", metav1.GetOptions{})
+	if err != nil {
+		log.WithError(err).Print("Error getting deployment to ping.", field.M{"app": "mysql"})
+	}
+	deploymentLabelSelector := deployment.Spec.Selector.MatchLabels
+	var podLabelSelector []string
+	for key, value := range deploymentLabelSelector {
+		podLabelSelector = append(podLabelSelector, key+"="+value)
+	}
+
+	pods, err := mdb.cli.CoreV1().Pods(mdb.namespace).List(metav1.ListOptions{
```

Comment:
Please use `kube.FetchPods()` here (https://github.com/kanisterio/kanister/blob/master/pkg/kube/workload.go#L172)

Reply:
Let's raise separate PR for this along with changes mentioned in https://github.com/kanisterio/kanister/pull/416/files#r347747536